### PR TITLE
PDP-1336-1340

### PR DESCRIPTION
**Pull request by akusingh-tibco**
Added quote in SA Name to accept integer, and commented the SA name in values.yaml so it doesn't creates SA on its own

PDP-1336 (tested)
```
AVXD1BF44:charts akusingh$ helm upgrade --install -n ankush dp-configure-namespace /Users/akusingh/Office/Github/tp-helm-charts/charts/dp-configure-namespace  --set global.tibco.dataPlaneId=ckt018e5d3vbed9qe6gg --set global.tibco.primaryNamespaceName=ankush --set global.tibco.serviceAccount=1234
Release "dp-configure-namespace" has been upgraded. Happy Helming!
NAME: dp-configure-namespace
LAST DEPLOYED: Thu Nov  9 08:45:23 2023
NAMESPACE: ankush
STATUS: deployed
REVISION: 2
TEST SUITE: None
NOTES:
# Copyright © 2023. Cloud Software Group, Inc.
# This file is subject to the license terms contained
# in the license file that is distributed with this file.
Namespace ankush validation for data plane id ckt018e5d3vbed9qe6gg label is successful.
Network policies creation is disabled by default.
To enable network policies creation, please use --set networkPolicy.create=true
Release name: dp-configure-namespace
To learn more about the release, try:
  $ helm status dp-configure-namespace -n ankush
  $ helm get all dp-configure-namespace -n ankush
```

PDP-1340 (tested)
```
AVXD1BF44:charts akusingh$ helm upgrade --install -n ankush dp-configure-namespace /Users/akusingh/Office/Github/tp-helm-charts/charts/dp-configure-namespace  --set global.tibco.dataPlaneId=ckt018e5d3vbed9qe6gg --set global.tibco.primaryNamespaceName=ankush
Release "dp-configure-namespace" does not exist. Installing it now.
Error: execution error at (dp-configure-namespace/templates/sa.yaml:13:11): global.tibco.serviceAccount is required
```

### CODE REVIEW

**Review by sasati-tibco — APPROVED** [review]

File: charts/dp-configure-namespace/Chart.yaml [hunk 1]
```diff
@@ -5,5 +5,5 @@ apiVersion: v2
 name: dp-configure-namespace
 description: A Helm chart for creating service account, cluster-role, cluster-role-binding, role-binding & network-policies
 type: application
-version: 1.0.15
+version: 1.0.16
```
Comment (nashtapu-tibco) [inline charts/dp-configure-namespace/Chart.yaml:8]: Should it be changed to 17 now?

File: charts/dp-configure-namespace/values.yaml [hunk 2]
```diff
@@ -10,7 +10,7 @@ global:
     primaryNamespaceName: "dp-namespace"
```
Comment (nashtapu-tibco) [inline charts/dp-configure-namespace/values.yaml:10]: comment this and dataPlaneId as well.

**Review by nashtapu-tibco — CHANGES_REQUESTED** [review]
Some comments.

Please test with helm lint and template. Also, try using digits as service account name.

File: charts/dp-configure-namespace/values.yaml (hunk 2 above)
Comment (akusingh-tibco) [reply charts/dp-configure-namespace/values.yaml:10]: Done

**Pull request merged**